app/services/hawkular_proxy_service.rb

```diff
@@ -33,7 +41,15 @@ def data(query)
   def metric_definitions
     tags = @params['tags'].blank? ? nil : JSON.parse(@params['tags'])
     tags = nil if tags == {}
-    client.gauges.query(tags).compact.map { |m| m.json if m.json }.sort { |a, b| a["id"].downcase <=> b["id"].downcase }
+
+    if @params['type'] == 'both'
```

Comment:
@yaacov `both`? This doesn't look like something you would use in an API.
When the `type` parameter is missing you would query all types.